src/plugins/discover/public/application/main/components/sidebar/lib/get_field_list.ts

```diff
@@ -73,7 +73,7 @@ export function getTextBasedQueryFieldList(
         name: column.name,
         type: column.meta?.type ?? 'unknown',
         esTypes: column.meta?.esType ? [column.meta?.esType] : undefined,
-        searchable: false,
+        searchable: true,
```

Comment:
I think we check `filterable` to decide whether to show filter buttons. Where do we check `searchable`?

Reply:
If you check the code the filterable is dependent on searchable

```
  public get filterable() {
    return (
      this.name === '_id' ||
      this.scripted ||
      ((this.spec.indexed || this.searchable) && this.kbnFieldType.filterable)
    );
  }
```
we are doing this in other places too and we used this flag to disable the filtering